adafruit_hts221.py

```diff
@@ -65,21 +71,22 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(
+        cls, value_tuples: Sequence[Tuple[str, int, Union[int, str]]]
+    ) -> None:
         """creates CV entries"""
-        cls.string = {}
-        cls.lsb = {}
+        cls.contents = {}
+        cls.label = {}
 
         for value_tuple in value_tuples:
-            name, value, string, lsb = value_tuple
+            name, value, label = value_tuple
             setattr(cls, name, value)
-            cls.string[value] = string
-            cls.lsb[value] = lsb
+            cls.label[value] = label
 
     @classmethod
-    def is_valid(cls, value):
+    def is_valid(cls, value: int) -> bool:
         """Returns true if the given value is a member of the CV"""
-        return value in cls.string
+        return hasattr(cls, value)
```

Comment:
You want:
```suggestion
        return value in cls.label
```
This is testing a value, not a name.
```py
hasattr("RATE_12_5_HZ", Rate)
```
versus:
```py
Rate.is_valid(3.1415)
```